### ISSUE TITLE
Performance Evaluation

### ISSUE THREAD

**Issue by SherlockHua1995** [ZhengPeng7/SANet-Keras#1]
HELLO, thanks for your open-source.
How about your recurrent performance?
Is this close to the result that paper mentioned?

**Comment by SherlockHua1995** [comment]
And would you plz offer the trained model?
Merry Christmas~

**Comment by ZhengPeng7** [comment]
> And would you plz offer the trained model?
> Merry Christmas~

Hi, very sorry for the late reply, I don't know why I didn't receive any notification from github in any method....
Back to your the topic, I did have trained the model, but the performance wasn't good enough, even quite a bit lower than the CSRnet on ShanghaiTech Dataset. However, the results are on a computer in my lab, and I've been home for Chinese New Year, so I can't give that to you now.
BTW, we tried to contact the author of original paper, but he only agreed to give us the model python file in pytorch, and the network structure was different from their original paper in about 4 key parts. So, perphaps the authors did made it, but I will keep my suspicion on the result untill there is any open-source code approached the performance in the original paper(BIGKnight is my friend, we did that together but both failed to reach the performance in the paper).
But, if you have any question on SANet, just contact me, always welcome.
